### PR TITLE
Run the utility on all maps

### PR DESCRIPTION
**Pull request by abcdefg30**
- Let the utility reorder stuff
- Let the utility remove spaces and tabs at the end of some lines
- Removed the `NOHQ` hack from `gdi03`
- Removed since #6964 redundant `MustBeDestroyed:` trait from `^CivInfantry` in `soviet01`

### CODE REVIEW

File: mods/ra/maps/soviet-01/map.yaml [hunk 1]
```diff
@@ -625,10 +625,6 @@ Rules:
 			GenericStancePrefix: false
 			GenericVisibility: Enemy
 			ShowOwnerRow: false
-	^CivInfantry:
-		MustBeDestroyed:
-	^CivBuilding:
-		MustBeDestroyed:
```
Comment (Mailaender) [inline mods/ra/maps/soviet-01/map.yaml]: You need to keep this as it won't inherit it from ^Building like ^CivInfantry does from ^Infantry.

Comment (abcdefg30) [inline mods/ra/maps/soviet-01/map.yaml]: Fail, fixed.

**Comment by pchote** [comment]
I ran the upgrader again on top of this pr, and found two more fixes you should include:
* gdi04a has stray whitespace at the end of a line.
* gdi05b's map.bin is saved in the old format.

**Comment by pchote** [comment]
:+1: after those two fixes.

**Comment by abcdefg30** [comment]
Updated with the new binary.
The issue in `nod04a` has already been covered by #7475.

**Comment by Mailaender** [comment]
gdi03 and soviet01 still work as promised. :white_check_mark:

**Pull request merged**